ipa-core/src/protocol/hybrid/step.rs

```diff
@@ -0,0 +1,6 @@
+use ipa_step_derive::CompactStep;
+
+#[derive(CompactStep)]
+pub(crate) enum HybridStep {
+    VerifyCiphertextUniqueness,
```

Comment:
doesn't seem to be used anywhere?

Reply:
you're right. originally I was going to pass it into a separate function and call `ctx.narrow`, but I dropped that when I changed the approach. is it useful to set the step here, or should I just drop that?

Reply:
If you plan to use it soon, we can keep it, otherwise it is probably better to remove it